frontends/main/next.config.js

```diff
@@ -57,6 +57,46 @@ const nextConfig = {
     ]
   },
 
+  async headers() {
+    return [
+      /* This is intended to target the base HTML responses. Some are dynamically rendered,
+       * so Next.js instructs no-cache, however we are currently serving public content that
+       * is cacheable. Excludes everything with a file extension so we're matching only on routes.
+       */
```

Comment:
My suggestion is mention RSC here, too.

```suggestion
      /* This is intended to target the base HTML responses and streamed RSC
       * content. Some routes are dynamically rendered, so NextJS by default
       * sets no-cache. However we are currently serving public content that is
       * cacheable.
       * 
       * Excludes everything with a file extension so we're matching only on routes.
       */
```